Makefile

```diff
@@ -64,7 +67,7 @@ cuda: $(CUDA_DEPS)  ## Build container for NVidia CUDA
 # The base container uses uids beyond 65535. Rootless builds may not work
 # unless the user account has extended subordinate ids up to 2**24 - 1.
 .PHONY: hpu
-hpu: $(HPU_DEPS)  ## Build container for Intel Gaudi HPU
+hpu: check-engine $(HPU_DEPS)  ## Build container for Intel Gaudi HPU
```

Comment:
Please move `check-engine` to the end of the dependency list. The first item `$<` has to be the container file.